### DISCUSSION TITLE
AOSP 12.0 v400.c

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#2062] [General]
This is a pre-release. This is not considered daily-drivable, though your mileage may vary.
As you'll notice, there is no A-only variant at the moment. It is still undecided whether A-only variants will later appear.

When you report issues in the bug tracker, don't forget to mention which GSI version you're using.

If you want unshared_blocks/read-write system, use vndklite variants. Non-vndklite devices are capable of running vndklite variants.

Known bugs:
- On various Mali devices (Mediatek, Samsung), there are graphical glitches at many places. You can workaround this issue by setting debug.sf.latch_unsignaled property to 0 with slower performances
- Moto One Action doesn't boot
- VoLTE/IMS is broken on Mediatek devices
- Alternate audio policy breaks volume on Samsung devices
- Pixel 1/2  have no sound in-call

Changes compared to v400.a:
- Fix SU
- Enable gapps and gogapps variants

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v400.c'>AOSP 12.0 v400.c</a>.</em>

**Comment by ibsondugrandb** [comment]
Bonjour , excellent travail j'utilise un infinix mediatek hot 8 x650b le GSI  "system-squeak-arm32_binder64-ab-gogapps.img.xz" mais l'alarme se plante à chaque que je veux planifier une heure de réveille. 
j'ai remarqué cette version les bouton de geste de navigations sont absents sauf celui à 3 boutons par défaut. et puis lorsque je change de fond d'écran Ya par la fonction Material You  qui génère le fond de couleur sur les tuiles. 
il n'y a pas de options widget aussi dans cette version. Merci encore pour votre travail. je suis en Afrique précisément au Burkina Faso et j'adore votre travail.

**Comment by tnolte1** [comment]
Excellent! Works great on my Blackview A80 Plus.
What is the recommended way to hide root from apps, e.g. from banking apps? I tried Phh Settings -> Securize which bricked my phone and also suhide and "Hide my toot" did not work.

**Reply by nasim7500** [reply]
Sir please build this Rom for A-only device,

Thank you so much sir for your effort

**Reply by Jubscleisson** [reply]
> Sir please build this Rom for A-only device,
> 
> Thank you so much sir for your effort 

if your device is SAR (system-as-root) you can probably use GSI AB :)

**Comment by rvtom1970** [comment]
Redmi not 8 pro, Mediatek, arm 64 ab, vendor miui_BEGONIAGlobal_V12.5.5.0.RGGMIXM_0e4a06b63a_11.0. Communication does not work at all.
[2021-11-06-15-03-29.txt](https://github.com/phhusson/treble_experimentations/files/7490477/2021-11-06-15-03-29.txt)

**Reply by dkostarnov** [reply]
автор на русском не говорит)

**Comment by dkostarnov** [comment]
Redmi 8a A32-Binder64 vdnk goapps
Cant enter to TWRP
Dont work "recent apps" button
Dont work gestures

Strong lags in the "Contacts" application from Google
Twitter feed lags

**Comment by kayuz6** [comment]
UmiDigi F1 connections don't work and no status bar on the top.

**Comment by B83C** [comment]
A-only variant please :)

**Comment by Chmax3008** [comment]
On the tablet huawei mediapad 10.8 lte cmr Al-09 installed, the camera works from a third-party application, the root works, bluetooth and wifi too. I have problems with graphics, the processor is kirin (Ps wrote to the phh mail and made a little donation for tea) Thanks 👍😊

**Comment by AlexPashenko** [comment]
mi11 everything works fine, except for one, the fingerprint scanner. is it possible to fix it?

**Comment by phhusson** [comment]
Yes it's possible. Ill try fixing it for Mi9, hopefully it'll fix it for Mi11. If it doesn't, it'll be hard for me unless I get one in my hand. (Hint: it's easier for me to get one if I need it if I get enough donations)

27 nov. 2021 13:22:46 AlexPashenko ***@***.***>:

**Reply by AlexPashenko** [reply]
I hope you will succeed. This is the only thing that doesn't work. If only there was face unlocking, it would be possible not to pay attention to the fingerprint😁

**Comment by nation888** [comment]
Device Oukitel K9 (Mediatek): Tried the system-squeak-arm64-ab-gapps and ran into a certified play services error for the first time. Yes my device is registered and then re-registered. Then tried to install Magisk to hide root, but was unsuccessful in removing SU from xbin. The system folder is read only and tried to remount rw folder using twrp -- Chmod 755, twrp terminal commands, PC adb commands and solid explorer. All other functions seem to work, except Gapps. History: I have flashed several GSI and flashed Gapps separately every since Lineage 15. This one won't let me mount system partition to resize and/or flash gapps. Anyone else having this certification error?

**Reply by Julinho55k** [reply]
is there how I can remove the root su and put the magisk without twrp?

**Reply by kkazakov** [reply]
I'm having the same certification error. It just won't let me pass and nothing google- related works.

**Comment by nation888** [comment]
I have installed Android 10/11 gsi without cert error. Must be a security feature in the Android 12 and gapps. This is a clue to research.  Regards, Trevor W. From: Krasimir KazakovSent: December 2, 2021 1:30 AMTo: phhusson/treble_experimentationsCc: nation888; CommentSubject: Re: [phhusson/treble_experimentations] AOSP 12.0 v400.c (Discussion #2062) I'm having the same certification error. It just won't let me pass and nothing google- related works.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by dankl** [comment]
My Cubot king mini works mostly fine. 

The mobile data connection is not working. The APN config was definitive right, but the phone was not able to establish a connection. I did a roll back to AOS 11.0 v313, which is working fine.

**Reply by devrieshh** [reply]
I have a cubot king kong mini2,do you now why the square menu sighn doesnt work?(triangle - point - square)
i have installed AOSP 11.0 v313 
also the sound is not hard enough when you talk calling to someone

**Comment by nation888** [comment]
My Oukitel K9 works perfectly so far except the certification play services
error. Yes I have registered. Also I have tried to flash magisk 23 before
booting. System partition is read only, tried to remount rw didn't work,
tried no verity just couldnt get it to work. So I thought it was Google
policy I installed Bliss dream Android 10 GSI and works perfectly. Does
anyone have ideas on solving this issue?  Is there a new twrp I could use?
As this version is aged.
